### PR TITLE
Update apps to work with 0.0.53rc

### PR DESCRIPTION
**Pull request by yanxi0830**
# What does this PR do?
- fix llama-stack-apps

## Feature/Issue validation/testing/test plan

```
$ llama stack build --template together --image-type conda
$ llama stack run ~/.llama/distributions/llamastack-together/together-run.yaml
```

```
python -m examples.agents.client localhost 5000
python -m examples.agents.hello localhost 5000
python -m examples.agents.inflation localhost 5000
python -m examples.agents.podcast_transcript localhost 5000
python -m examples.agents.rag_as_attachments localhost 5000
python -m examples.agents.rag_with_memory_bank localhost 5000
```

## Sources

Please link relevant resources if necessary.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/meta-llama/llama-stack-apps/blob/main/CONTRIBUTING.md#pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

Thanks for contributing 🎉!

### CODE REVIEW

File: examples/agents/client.py [hunk 1]
```diff
@@ -37,12 +36,16 @@ async def run_main(
 
     available_shields = [shield.identifier for shield in client.shields.list()]
     if not available_shields:
-        print(f"No available shields. Disable safety.")
+        print("No available shields. Disable safety.")
     else:
         print(f"Available shields found: {available_shields}")
 
+    available_models = [model.identifier for model in client.models.list()]
+    selected_model = available_models[0]
```
Comment (ashwinb) [inline examples/agents/client.py:44]: what if there is no available model? it should error out

File: examples/agents/hello.py [hunk 2]
```diff
@@ -22,12 +22,15 @@ async def run_main(host: str, port: int, disable_safety: bool = False):
 
     available_shields = [shield.identifier for shield in client.shields.list()]
     if not available_shields:
-        print(f"No available shields. Disable safety.")
+        print("No available shields. Disable safety.")
     else:
         print(f"Available shields found: {available_shields}")
+    available_models = [model.identifier for model in client.models.list()]
+    selected_model = available_models[0]
```
Comment (ashwinb) [inline examples/agents/hello.py:29]: same thing

File: examples/agents/inflation.py [hunk 3]
```diff
@@ -23,12 +23,15 @@ async def run_main(host: str, port: int, disable_safety: bool = False):
 
     available_shields = [shield.identifier for shield in client.shields.list()]
     if not available_shields:
-        print(f"No available shields. Disable safety.")
+        print("No available shields. Disable safety.")
     else:
         print(f"Available shields found: {available_shields}")
+    available_models = [model.identifier for model in client.models.list()]
+    selected_model = available_models[0]
```
Comment (ashwinb) [inline examples/agents/inflation.py:30]: same thing, this likely needs to be another common file? but we hate common files in the examples :/

**Review by ashwinb — APPROVED** [review]

File: examples/agents/inflation.py (hunk 3 above)
Comment (yanxi0830) [reply examples/agents/inflation.py:30]: ugh yeah, I really hate the individual scripts here, too many updates whenever SDK changes. maybe need to move to a common llama-stack-client helper as follow up

**Pull request merged**